src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/VedtakOgBeregningBarnetilsyn.tsx

```diff
@@ -17,40 +24,66 @@ const Container = styled.div`
+    const { behandling } = useBehandling();
     const { vedtak } = useVedtak();
 
+    const [vilkårsoppsummering, settVilkårsoppsummering] =
+        useState<Ressurs<Vilkårsoppsummering>>(byggTomRessurs());
     const [typeVedtak, settTypeVedtak] = useState<TypeVedtak | undefined>();
 
+    const hentVilkåroppsummering = useCallback(() => {
+        request<Vilkårsoppsummering, null>(`/api/sak/vilkarsoppsummering/${behandling.id}`).then(
+            settVilkårsoppsummering
+        );
+    }, [behandling, request]);
+
+    useEffect(() => {
+        hentVilkåroppsummering();
+    }, [hentVilkåroppsummering]);
+
     useEffect(() => {
         if (vedtak.status === RessursStatus.SUKSESS) {
             settTypeVedtak(vedtak.data.type);
         }
     }, [vedtak]);
 
     return (
-        <Container>
-            <VelgVedtakResultat typeVedtak={typeVedtak} settTypeVedtak={settTypeVedtak} />
-            <DataViewer response={{ vedtak }}>
-                {({ vedtak }) => {
-                    switch (typeVedtak) {
-                        case TypeVedtak.INNVILGELSE:
-                            return (
-                                <InnvilgeVedtak lagretVedtak={vedtak as InnvilgelseBarnetilsyn} />
-                            );
-
-                        case TypeVedtak.AVSLAG:
-                            return <AvslåVedtak vedtak={vedtak as AvslagBarnetilsyn} />;
-
-                        case undefined:
-                            return null;
-
-                        default:
-                            return <p>Ikke implementert</p>;
-                    }
-                }}
+        <>
+            <DataViewer response={{ vedtak, vilkårsoppsummering }}>
+                {({ vedtak, vilkårsoppsummering }) => (
+                    <Container>
+                        <Panel tittel="Vedtak">
+                            <HStack gap="10">
+                                <OppsummeringVilkår vilkårsoppsummering={vilkårsoppsummering} />
+                                <Skillelinje />
+                                <HStack gap="16">
+                                    <VelgVedtakResultat
+                                        typeVedtak={typeVedtak}
+                                        settTypeVedtak={settTypeVedtak}
+                                    />
+                                    {typeVedtak === TypeVedtak.AVSLAG && (
+                                        <AvslåVedtak vedtak={vedtak as AvslagBarnetilsyn} />
+                                    )}
+                                </HStack>
+                            </HStack>
+                        </Panel>
+
+                        {typeVedtak === TypeVedtak.INNVILGELSE && (
+                            <InnvilgeBarnetilsyn
+                                lagretVedtak={vedtak as InnvilgelseBarnetilsyn}
+                                vilkårsvurderteBarn={vilkårsoppsummering.passBarn}
+                            />
+                        )}
```

Comment:
Burde komponenten for avslag og innvilgelse være litt mer gruppert i koden? 
Nå ligger avslag nestet inne i noen stacks, mens innvilgelse ligger helt utenfor. Eller er det noen spesiell grunn til at det er sånn?

Reply:
Ser nå på bildene at avslag havner til høyre av oppsummeringen

Reply:
Ja:/ 
Koden ble ofret for å få til designet som var ønskelig. Hvis du har forslag til hvordan det kan bli bedre blir jeg veldig glad! Likte egt bedre switch som var før